[2023-05-14T17:56:44.739+0000] {taskinstance.py:1084} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: get_todays_gamesv2.call_games manual__2023-05-14T17:56:41.615321+00:00 [queued]>
[2023-05-14T17:56:44.751+0000] {taskinstance.py:1084} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: get_todays_gamesv2.call_games manual__2023-05-14T17:56:41.615321+00:00 [queued]>
[2023-05-14T17:56:44.751+0000] {taskinstance.py:1282} INFO - 
--------------------------------------------------------------------------------
[2023-05-14T17:56:44.751+0000] {taskinstance.py:1283} INFO - Starting attempt 1 of 1
[2023-05-14T17:56:44.751+0000] {taskinstance.py:1284} INFO - 
--------------------------------------------------------------------------------
[2023-05-14T17:56:44.780+0000] {taskinstance.py:1303} INFO - Executing <Task(PythonOperator): call_games> on 2023-05-14 17:56:41.615321+00:00
[2023-05-14T17:56:44.790+0000] {standard_task_runner.py:55} INFO - Started process 2981 to run task
[2023-05-14T17:56:44.795+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'get_todays_gamesv2', 'call_games', 'manual__2023-05-14T17:56:41.615321+00:00', '--job-id', '1426', '--raw', '--subdir', 'DAGS_FOLDER/todays_gamesv2.py', '--cfg-path', '/tmp/tmpmzlwvgs4']
[2023-05-14T17:56:44.796+0000] {standard_task_runner.py:83} INFO - Job 1426: Subtask call_games
[2023-05-14T17:56:44.874+0000] {task_command.py:388} INFO - Running <TaskInstance: get_todays_gamesv2.call_games manual__2023-05-14T17:56:41.615321+00:00 [running]> on host f7ab54451666
[2023-05-14T17:56:44.964+0000] {taskinstance.py:1511} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=get_todays_gamesv2
AIRFLOW_CTX_TASK_ID=call_games
AIRFLOW_CTX_EXECUTION_DATE=2023-05-14T17:56:41.615321+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-05-14T17:56:41.615321+00:00
[2023-05-14T17:56:45.192+0000] {logging_mixin.py:137} INFO - {0: {'game_id': 718180, 'game_datetime': '2023-05-14T17:35:00Z', 'game_date': '2023-05-14', 'game_type': 'R', 'status': 'In Progress', 'away_name': 'Tampa Bay Rays', 'home_name': 'New York Yankees', 'away_id': 139, 'home_id': 147, 'doubleheader': 'N', 'game_num': 1, 'home_probable_pitcher': 'Clarke Schmidt', 'away_probable_pitcher': 'Zach Eflin', 'home_pitcher_note': '', 'away_pitcher_note': '', 'away_score': 1, 'home_score': 0, 'current_inning': 2, 'inning_state': 'Top', 'venue_id': 3313, 'venue_name': 'Yankee Stadium', 'national_broadcasts': ['MLB.tv Free Game'], 'series_status': 'NYY leads 2-1', 'summary': '2023-05-14 - Tampa Bay Rays (1) @ New York Yankees (0) (Top of the 2nd)'}}
[2023-05-14T17:56:45.193+0000] {taskinstance.py:1775} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 175, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 192, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/opt/airflow/dags/todays_gamesv2.py", line 44, in _call_games
    game_time = transform_time(games_dict['game_datetime'])
KeyError: 'game_datetime'
[2023-05-14T17:56:45.213+0000] {taskinstance.py:1326} INFO - Marking task as FAILED. dag_id=get_todays_gamesv2, task_id=call_games, execution_date=20230514T175641, start_date=20230514T175644, end_date=20230514T175645
[2023-05-14T17:56:45.229+0000] {standard_task_runner.py:105} ERROR - Failed to execute job 1426 for task call_games ('game_datetime'; 2981)
[2023-05-14T17:56:45.255+0000] {local_task_job.py:212} INFO - Task exited with return code 1
[2023-05-14T17:56:45.296+0000] {taskinstance.py:2585} INFO - 0 downstream tasks scheduled from follow-on schedule check
